react/Table/Virtualized/Dnd/TableRow.jsx

```diff
@@ -17,8 +17,9 @@ const TableRow = ({ item, context, ...props }) => {
     canDrag: canDragProps,
     dragId
   } = dragProps
+  const { isCut } = props
```

Comment:
```suggestion
  const { isDimmed } = props
```

I would use other names not to mix Drive and Cozy-UI terms:
- isInactive
- isDimmed
- hasReducedOpacity